lib/ews/soap/builders/ews_builder.rb

```diff
@@ -1190,10 +1190,9 @@ def dispatch_item_id!(iid)
       when :item_id
         item_id!(item)
       when :occurrence_item_id
-        occurrence_item_id!(
-          item[:recurring_master_id], item[:change_key], item[:instance_index])
+        occurrence_item_id!(item)
```

Comment:
Fix dispatch_item_id method, so that we can correctly call the GetItems method, which requires us to pass in several parameters and currently it was not working. This issue has been fixed in a later version of viewpoint, with the exact same changes
https://github.com/WinRb/Viewpoint/blob/master/lib/ews/soap/builders/ews_builder.rb#L1182